html/semantics/popovers/popover-attribute-all-elements.html

```diff
@@ -33,7 +33,15 @@
       assertNotAPopover(element);
     }, `A <${tag}> element should *not* behave as a popover.`);
   });
-
+  elementsThatDontRender.forEach(tag => {
+    test((t) => {
+      const element = document.createElement(tag);
+      element.setAttribute('popover','auto');
+      document.body.appendChild(element);
+      t.add_cleanup(() => element.remove());
+      assert_equals(window.getComputedStyle(element).display, "none", "Element isn't rendered");
+    }, `A <${tag} popover> element should *not* behave as a popover.`);
```

Comment:
It still has popover behaviors like matching `:open`/`:closed`, so this is probably more accurate:

```suggestion
    }, `A <${tag} popover> element should not be rendered`);
```